ats/test/HelloATS/create_test_ats.py

```diff
@@ -1,21 +1,22 @@
-#!/usr/apps/ats/7.0.1/bin/python
+#!/usr/bin/env python3
 """Create a test.ats file."""
 import itertools
 import os
+import sys
 
 
 def get_file_header():
     """Returns ATS test file header. Contains imports, glue, and checker."""
     ats_log_checker_path = os.path.join(
         os.path.dirname(os.path.abspath(__file__)), 'ats_check_log.py')
     if not os.path.isfile(ats_log_checker_path):
-        exit(1)
+        sys.exit(1)
 
-    ofp_header = """import os
+    ofp_header = f"""import os
 glue(independent=True)
 glue(keep=True)
-my_checker = '%s'
-""" % ats_log_checker_path
+my_checker = '{ats_log_checker_path}'
+"""
```

Comment:
Just to clarify, it's okay that these changes break Python 2 support?

Reply:
Actually using f strings breaks support for anything prior to Python 3.6. 😄 
In this small script it's fine since "create_test_ats.py" is not referenced elsewhere and the script starts with "#!/usr/bin/env python3".
Also I feel like it's a safe to assume that Python 3 users are using at least 3.6 since that was released about 4.5 years ago.

Reply:
Ooo! 3.6! That's later than I thought! I agree, it's widely available and shouldn't be a problem